### PR TITLE
OSDOCS-12028: DOcumented the 4.14.37 z-stream notes

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.14

Issue:
[OSDOCS-12028](https://issues.redhat.com/browse/OSDOCS-12028)

Link to docs preview:
[4.14.37](https://81940--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-37_release-notes)

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3340,6 +3340,28 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.37
+[id="ocp-4-14-37_{context}"]
+=== RHSA-2024:6689 - {product-title} 4.14.37 bug fix and security update
+
+Issued: 19 September 2024
+
+{product-title} release 4.14.37, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6689[RHSA-2024:6689] advisory. No RPM packages exist for this update.
```
Comment (opayne1) [inline release_notes/ocp-4-14-release-notes.adoc:3349]:
```suggestion
{product-title} release 4.14.37, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6689[RHSA-2024:6689] advisory. There are no RPM packages for this update.
```

Not sure if you all update it, but iirc this is what we have said in the past for no RPM updates.

**Review by opayne1 — COMMENTED** [review]
One small comment but otherwise lgtm!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc]: :heavy_check_mark:

**Pull request merged**